### PR TITLE
Iphone X queries

### PR DESCRIPTION
**Pull request by OlegBaras**

### CODE REVIEW

File: src/components/css/HeaderNav.css [hunk 1]
```diff
@@ -2,19 +2,14 @@ nav {
...
 }
 
 .nav-links {
-  padding-left: 2rem;
   display: flex;
-  justify-content: space-around;
-  align-items: center;
   list-style: none;
   gap: 1rem;
+  flex-grow: 1;
```
Comment (aurimas-darguzis) [inline src/components/css/HeaderNav.css:12]: jei sita isimsi - UI pasikeis?

File: src/components/css/HeaderNav.css [hunk 2]
```diff
@@ -47,48 +42,43 @@ nav {
...
   }
 
   .nav-links li {
-    font-size: 16px;
+    font-size: 18px;
   }
 
   .nav-links li::after {
     content: none;
     /* or display: none - what is the difference */
```
Comment (aurimas-darguzis) [inline src/components/css/HeaderNav.css:75]: remove comment

File: src/components/css/LanguageSelector.css [hunk 3]
```diff
@@ -2,8 +2,9 @@
   display: flex;
   align-self: flex-end;
   justify-content: flex-end;
-  transform: translateY(-4.7vh) translateX(-1rem);
-  /* border: 2px solid purple; */
+  /* transform: translateY(-4.7vh) translateX(-1rem); */
```
Comment (aurimas-darguzis) [inline src/components/css/LanguageSelector.css:5]: remove comment

File: src/components/css/LanguageSelector.css [hunk 4]
```diff
@@ -24,14 +25,18 @@
...
 @media all and (min-width: 375px) and (max-width: 768px) {
   .languageSelector {
-    width: 0;
+    /* width: 0;
     transform: translateY(-0.5rem) translateX(1.2rem);
     gap: 0.5rem;
     position: absolute;
     top: 1.1rem;
-    right: 1rem;
+    right: 1rem; */
```
Comment (aurimas-darguzis) [inline src/components/css/LanguageSelector.css:33]: remove comments

**Review by aurimas-darguzis — CHANGES_REQUESTED** [review]
few things to imrpove

File: src/components/css/HeaderNav.css (hunk 1 above)
Comment (OlegBaras) [reply src/components/css/HeaderNav.css:12]: isemiau, nepasikeite.

Comment (aurimas-darguzis) [reply src/components/css/HeaderNav.css:12]: Tai reiskias net ir nereikejo jo cia. Istrink tada. Nereikia kodo, kuris nieko nedaro

**Review by OlegBaras — COMMENTED** [review]
comments ammended

**Pull request merged**